mmcv/transforms/processing.py

```diff
@@ -400,3 +400,407 @@ def __repr__(self):
         repr_str += f'pad_val={self.pad_val}), '
         repr_str += f'padding_mode={self.padding_mode})'
         return repr_str
+
+
+@TRANSFORMS.register_module()
+class RandomFlip(BaseTransform):
+    """Flip the image & bbox & keypoints & segmentation map.
+
+    There are 3 flip modes:
+
+    - ``prob`` is float, ``direction`` is string
+
+        the image will be ``direction`` ly flipped with probability
```

Comment:
the image will be ``direction`` ly flipped with probability -> the image will be flipped on the given ``direction`` with probability
Also change that in line 419